semconv-incubating/src/main/java/io/opentelemetry/semconv/incubating/ContainerIncubatingAttributes.java

```diff
@@ -106,46 +91,11 @@ public final class ContainerIncubatingAttributes {
   public static final AttributeKeyTemplate<String> CONTAINER_LABEL =
       stringKeyTemplate("container.label");
 
-  /**
-   * Deprecated, use {@code container.label} instead.
-   *
-   * @deprecated Deprecated, use `container.label` instead.
-   */
-  @Deprecated
```

Comment:
What's going on with the removal of these deprecated attributes?

Reply:
I'll take a look.  Likely I've filtered them out in my JQ filter for some reason...

Reply:
It looks like all the experimental + deprecated attributes are not showing up here from the deprecated group in the registry, I'm investigating

Reply:
Ah right - If you recall the `client.id` vs. `client_id` debacle, the solution that was decided upon was to stop generating *deprecated* + *experimental* attributes in codegen.

Effectively, these are attributes that not only never stabilized, but also will never stabilize.

Including those types will cause a conflict in codegen between two generated code names.  We only guarantee uniqueness of generated code for the set of (stable attributes and non-deprecated experimental attributes).

So this is actually updating Java codegen here.  I can continue generating the deprecated experimental attributes, but two things:

(1) You'll need to carry any namespace-related conflict resolution that causes in the java codegen going forward.
(2) We likely want some way to denote "This is deprecated for the Stable thing" vs. "This is deprecated and never use it again".

cc @lmolkova to correct me if I confused anything on this resolution.

Reply:
How to resolve client.id vs client_id - https://github.com/open-telemetry/semantic-conventions/issues/1031#issuecomment-2173887975 - manually drop `messaging.client_id`.

Existing jinja and build-tools solution implements it already and here's the weaver-based approach in python
- [declare a list of excluded ones](https://github.com/open-telemetry/opentelemetry-python/blob/889f7df41ecca926c7847f3c0913b7b9bbc0dd62/scripts/semconv/templates/registry/weaver.yaml#L7)
- [exclude them in jinja](https://github.com/open-telemetry/opentelemetry-python/blob/889f7df41ecca926c7847f3c0913b7b9bbc0dd62/scripts/semconv/templates/registry/semantic_attributes.j2#L22) or in JQ (in python we decided to generate them as comments for transparency) 


WRT if we should generate deprecated+experimental attributes - we should. The motivation - dependency version conflicts - there is always a risk that semconv artifact version is resolved to something you don't expect. 

If experimental+deprecated attributes are not generated, it could mean that two parts of app that use schema_url vN and schema_url N+1 can become incompatible and can't coexist since there is no semconv artifact that contains both.

By generating experiemental+deprecated we make it ~~much easier~~ possible to update different parts of user app independently

Reply:
Ok, this is now fixed.